src/components/CompareResults/ResultsTitle.tsx

```diff
@@ -0,0 +1,87 @@
+import { useState } from 'react';
+
+import EditIcon from '@mui/icons-material/EditOutlined';
+import { Input } from '@mui/material';
+import { style } from 'typestyle';
+
+import useRawSearchParams from '../../hooks/useRawSearchParams';
+import { FontsRaw, Colors } from '../../styles';
+
+export const ResultsTitle = () => {
+  const [searchParams, updateSearchParams] = useRawSearchParams();
+  const [resultsTitle, setResultsTitle] = useState(
+    searchParams.get('title') || 'Results',
+  );
+  const [isEditing, setIsEditing] = useState(false);
+
+  const styles = {
+    title: style({
```

Comment:
Also the text and the icon will need to be vertically aligned. If you use a `display :flex` or `display: inline-flex` + `align-items: center` this should make them aligned.